src/cli/commands/global.js

```diff
@@ -32,7 +32,10 @@ class GlobalAdd extends Add {
 const path = require('path');
 
 async function updateCwd(config: Config): Promise<void> {
-  await config.init({cwd: config.globalFolder});
+  await config.init({
+    cwd: config.globalFolder,
+    binLinks: true,
```

Comment:
> Shouldn't we take the value of config.binLinks instead of hard coding the value of binLinks to true.

That would be awesome but for some reason `config.binLinks` is false here. Since this is a major blocker for lots of people so I decided to fix it with `true` for the time being.

> What about all other option, config.init will set all other Boolean options to false, or to the default value from constants.

I tried what you are trying to do but it will break some other things. For example `prefix`. I have already asked @kittens about it. We might need some refactoring here.